test/assert-file-to-not-exist-test.js

```diff
@@ -7,8 +8,13 @@ describe('assert-file-to-not-exist', function() {
   });
 
   it('should throw if file exists', function() {
-    expect(function() {
+    var err = catchError(function() {
```

Comment:
I guess we could override the throw property to handle a function argument. I'll have a look.